modules/reference/pages/admin/show-tomcat-logs.adoc

```diff
@@ -1,7 +1,10 @@
 [[ref.webui.admin.logs]]
 = Show Tomcat Logs
 
-Here the {productname} admin user has access to the Tomcat log file located at [path]``/var/log/rhn/rhn_web_ui.log``.
-No {rootuser} privileges are required.
+The menu:Admin[Show Tomcat Logs] section shows the Apache Tomcat log file.
+You can also view the Tomcat log from the command prompt at [path]``/var/log/rhn/rhn_web_ui.log``.
 
-image::admin_show_tomcat_logs.png[scaledwidth=80%]
+[NOTE]
+====
+The menu:Admin[Show Tomcat Logs] section is only available if you are logged in with a {productname} administrator account.
```

Comment:
Wondering whether active voice would be better?

BTW, the SUMA start page uses the term "Sign In".

Reply:
Using the "most important thing first" rule, I think this works better. I'm not too strongly sold either way, though. Edited to "signed in" though :)